src/cogent3/core/tree.py

```diff
@@ -1598,7 +1599,74 @@ def rooted_with_tip(self, outgroup_name):
         tip = self.get_node_matching_name(outgroup_name)
         return tip.parent.unrooted_deepcopy()
 
-    def lin_rajan_moret(self, tree2) -> float:
+    def tree_distance(self, other: TreeNode, method: str | None = None) -> int:
+        """Return the specified tree distance between this and another tree.
+
+        Defaults to the Lin-Rajan-Moret distance on unrooted trees.
+        Defaults to the Matching Cluster distance on rooted trees.
+
+        Parameters
+        ----------
+        other: TreeNode
+            The other tree to calculate the distance between.
+        method: str | None
+            The tree distance metric to use.
+
+            Options are:
+            "rooted_robinson_foulds": The Robinson-Foulds distance for rooted trees.
+            "unrooted_robinson_foulds": The Robinson-Foulds distance for unrooted trees.
+            "matching_cluster": The Matching Cluster distance for rooted trees.
+            "lin_rajan_moret": The Lin-Rajan-Moret distance for unrooted trees.
+            "rrf": An alias for rooted_robinson_foulds.
+            "urf": An alias for unrooted_robinson_foulds.
+            "mc": An alias for matching_cluster.
+            "lrm": An alias for lin_rajan_moret.
+            "rf": The unrooted/rooted Robinson-Foulds distance for unrooted/rooted trees.
+            "matching": The Lin-Rajan-Moret/Matching Cluster distance for unrooted/rooted trees.
+
+            Default is "matching".
+
+        Returns
+        -------
+        int
+            the chosen distance between the two trees.
+
+        Notes
+        -----
+        The Lin-Rajan-Moret distance and Matching Cluster distance displays
+        superior statistical properties than the Robinson-Foulds distance on
+        unrooted and rooted trees respectively.
+
+        see: Bogdanowicz, D., & Giaro, K. (2013).
```

Comment:
For including citations, see the [numpydoc documentation](https://numpydoc.readthedocs.io/en/latest/format.html#references).